prep-html-syntax/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<style>
```

Comment:
You are missing your `<head>` tag with a `<title>` tag inside it. Also the `<style>` tags should be within the `<head>` tags as well.